tests/tools/fio/workload_test.go

```diff
@@ -0,0 +1,49 @@
+package fio
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/kopia/kopia/tests/testenv"
+)
+
+func TestWriteFiles(t *testing.T) {
+	r, err := NewRunner()
+	testenv.AssertNoError(t, err)
+
+	defer r.Cleanup()
+
+	relativeWritePath := "some/path/to/check"
+	writeSizeB := int64(3 * 1024 * 1024 * 1024) // 3 GiB
```

Comment:
Since the goal here is to quickly test whether or not `fio` does what we expect it to do, does it make sense generate a much smaller amount of data (in the order of a few MB) to speed things up.

Reply:
Yeah can do. It was returning errors for smaller transfers due to one of the default global settings, but that can be changed. I'll revisit